govc/vm/customize.go

```diff
@@ -98,6 +95,36 @@ Examples:
   govc vm.customize -vm VM -tz America/New_York NAME`
 }
 
+// Parse a string of multiple IPv6 addresses with optional netmask separated by space
+func parseIPv6Argument(argv string) (ipconf []types.BaseCustomizationIpV6Generator, err error) {
+	for _, substring := range strings.Split(argv, " ") {
```

Comment:
Can we make this comma delimited instead of space?  We use comma elsewhere for similar.